Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/EventRegistrationModule.kt

```diff
@@ -6,10 +6,15 @@ import de.rki.coronawarnapp.eventregistration.checkins.download.DownloadedCheckI
 import de.rki.coronawarnapp.eventregistration.checkins.download.FakeDownloadedCheckInsRepo
 import de.rki.coronawarnapp.eventregistration.checkins.qrcode.DefaultQRCodeVerifier
 import de.rki.coronawarnapp.eventregistration.checkins.qrcode.QRCodeVerifier
+import de.rki.coronawarnapp.eventregistration.events.ui.category.TraceLocationCategoryModule
 import de.rki.coronawarnapp.eventregistration.storage.repo.DefaultTraceLocationRepository
 import de.rki.coronawarnapp.eventregistration.storage.repo.TraceLocationRepository
 
-@Module
+@Module(
+    includes = [
+        TraceLocationCategoryModule::class,
+    ]
+)
```

Comment:
I think this one should go into `EventRegistrationUIModule `

Reply:
Yes, fixed in new commit.